### DISCUSSION TITLE
Why doesn the Firefox extension ask to read my browser history?

### DISCUSSION THREAD

**Discussion by louis77** [dailydotdev/daily#498] [General]
I'm wondering why daily.dev needs to read my browser history from the Firefox extension?

**Comment by idoshamun** [comment]
Hi @louis77,

This is optional permission that allows us to show you your most visited sites.
By default, it shouldn't ask for this permission, only if you want to enable the shortcuts.
In any way, we never send this information back to our servers. Everything is done on the client-side.
